plotting/operationalPlots.py

```diff
@@ -328,7 +330,7 @@ def plotSpatialFieldData(contourpacket, fieldpacket, prefix='', nested=True, mod
     levels = np.linspace(cbar_min, cbar_max, 35)  # draw 35 levels
     norm = mc.BoundaryNorm(levels, 256)
 
-    from plottingTools import  MidpointNormalize
+    from .plottingTools import  MidpointNormalize
```

Comment:
does this run as from .plottingtools import midpointNormalize?

Reply:
***This will have to be dealt with separately from this commit - we will be removing the entire plotting library from this merge and deal with it as a separate folder like we are doing with the submodules***